src/typings/react-native.d.ts

```diff
@@ -240,6 +240,9 @@ declare module 'react-native' {
         accessible?                      : boolean;
         onAcccessibilityTap?             : Function;
 
+        // desktop
+        ariaValueNow?                        : number;
```

Comment:
Actually, this one is not needed in this file at all, please have a look at ariaRoleDescription or ariaLabelledBy. They only present in ViewPropsShared in Types.ts and in the web implementation of the View.

Reply:
Removed